examples/app-teardown.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+DIR=`dirname "$0"`
+
+cd $DIR
+export FLASK_APP=app.py
+
+# Delete the database
+flask db destroy --yes-i-know
+
+# Delete indices
+flask index destroy --yes-i-know --force
+
+# clean environment
+[ -e "$DIR/instance" ] && rm $DIR/instance -Rf
+# [ -e "$DIR/static" ] && rm $DIR/static -Rf
```

Comment:
there are many other files to delete inside the static folder.
I'll add a line for every one, WDYT? (see https://github.com/inveniosoftware/invenio-deposit/pull/122/files#diff-d2d5e18fe9b7cce794aa5d22282806c3R15)